README.md

```diff
@@ -82,37 +82,54 @@ These are the empty repositories that have PRs waiting to be merged.
 
 ### rust-vmm
 
-- [vm-device](https://github.com/rust-vmm/vm-device): a virtual machine device
-  model crate.
-- [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
-  implementation for virtio primitives such as virtqueues and descriptor chain.
+- [mshv](https://github.com/rust-vmm/mshv): workspace for Microsoft Hypervisor
+  safe wrappers.
+- [vm-allocator](https://github.com/rust-vmm/vm-allocator): abstractions for
+  allocating resources needed by the VMM while running (such as GSI numbers,
+  MMIO ranges).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio): workspace for virtio
+  devices and related helpers.
 - [vmm-reference](https://github.com/rust-vmm/vmm-reference): reference VMM
   built solely with `rust-vmm` crates and minimal glue.
+- [vhost-device](https://github.com/rust-vmm/vhost-device): A workspace for
+  devices implemented using the
+  [vhost-user-backend](https://github.com/rust-vmm/vhost-user-backend).
 
 ### crates.io
 
+- [event-manager](https://crates.io/crates/event-manager): abstractions
+  for implementing event based systems.
 - [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
   to KVM generated using [bindgen](https://crates.io/crates/bindgen).
 - [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
   KVM API.
-- [vfio-bindings](https://crates.io/crates/vfio-bindings): Rust FFI
-  bindings for using the VFIO framework.
+- [linux-loader](https://crates.io/crates/linux-loader): parser and loader
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [seccompiler](https://github.com/rust-vmm/seccompiler): Linux seccomp-bpf
```

Comment:
I just noticed that we used to add the crates.io link in this section, instead of the github one, so I think we should continue doing that.